common/types.go

```diff
@@ -0,0 +1,45 @@
+package common
```

Comment:
Moving the types to the common folder is going to break packages using the types from this library (e.g. https://github.com/onflow/flow-cli/blob/master/internal/super/flix.go#L107). If they are moved to a common library then they need to be reexported out the main entry of the package.